.github/workflows/db-migrate.yml

```
@@ -0,0 +1,30 @@
+      - main
+
+jobs:
+  db-migrate:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code 🧾
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js ⚛️
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)